src/rate/bitmax.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import { asObject, asString } from 'cleaners'
+import {
+  type EdgeCorePluginOptions,
+  type EdgeRatePlugin
+} from 'edge-core-js/types'
+
+const asBitMaxTickerResponse = asObject({ data: asObject({ close: asString }) })
+
+export function makeBitMaxPlugin(opts: EdgeCorePluginOptions): EdgeRatePlugin {
+  const { io } = opts
+  const { fetchCors = io.fetch } = io
+
+  return {
+    rateInfo: {
+      pluginId: 'bitmax',
+      displayName: 'BitMax'
+    },
+
+    async fetchRates(pairsHint) {
+      const response = await fetchCors(
+        'https://bitmax.io/api/pro/v1/ticker?symbol=FIO/USDT'
+      )
+      const json = await response.json()
+      if (!response.ok || json.reason === 'DATA_NOT_AVAILABLE') {
+        // Return fixed rate if data is unavailable
+        return [
+          {
+            fromCurrency: 'FIO',
+            toCurrency: 'USDT',
+            rate: 0.001
+          }
+        ]
+      }
+      asBitMaxTickerResponse(json)
+      return [
+        {
+          fromCurrency: 'FIO',
+          toCurrency: 'USDT',
+          rate: Number(json.data.close)
```

Comment:
Really you want the avg between the bid and ask. The `close` will give the previous day closing price. 

In summary, use the avg of `data.bid[0]` and `data.ask[0]`

Reply:
"close" is the last traded price and doesn't seem related to a time period https://bitmax-exchange.github.io/bitmax-pro-api/#ticker